docs/guide/template-filters.md

```diff
@@ -0,0 +1,131 @@
+# Template filters
+
+## Nunjucks supplied filters
+The Nunjucks templating language comes with several [built-in filters](https://mozilla.github.io/nunjucks/templating.html#filters). These filters apply functions to template data before outputting content to the page. They are called with a pipe operator `|` and can take arguments.
+
+```markup
+<h1>{{ data.page.headline | replace("foo", "bar" ) | upper }}</h1>
+```
+
+In this example, a hypothetical page headline will be retrieved from the page data and then piped to the 'replace' filter, along with two arguments. This filter will scan the incoming data and replace any instances of ‘foo’ with 'bar'. The output of this filter will then pass the result to the upper function, which will return the input as an uppercase string before being output to the page.
+
+>Note: The order of filters can be significant, as they are applied sequentially to the input data.
```

Comment:
👋 Vuepress has nice note blocks like `::: tip` used elsewhere. The heading can be customized to "Note" as well. This saves block quotes for actual quotations.

https://v1.vuepress.vuejs.org/guide/markdown.html#custom-containers

Reply:
Changed.